R/tax_range_time.R

```diff
@@ -0,0 +1,178 @@
+#' Calculate the temporal range of fossil taxa
+#'
+#' A function to calculate the temporal range of fossil taxa.
+#'
+#' @param occdf \code{dataframe}. A dataframe of fossil occurrences. The
+#' dataframe should contain at least three columns: names of taxa (`character`)
+#' and the maximum (`numeric`) and minimum (`numeric`) age which constrain the
+#' age range (in millions of years) of the fossil occurrence. If the age range
```

Comment:
Do the values have to be in millions of years? Surely the unit you put in is the unit you get out. I'm just thinking that maybe some people who work with near-recent data might want to use the function. In this case, it might be worth renaming some of the variables, e.g. 'max_ma' -> 'max_time'.

Reply:
As above! We can modify this, but would require some fiddling with the plotting functionality (specifically, axis_geo).